red/Animal/scripts/barnyard.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
+
+use v5.14;
+use strict;
+use warnings;
+
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+
+our @ANIMALS = qw( Cow Horse Sheep Mouse );
```

Comment:
True, though I like to use it for package wide variables to make the scope immediately obvious.